src/renderer.rs

```diff
@@ -144,19 +157,33 @@ impl Renderer {
         );
         let shaders = shaders.await;
 
-        let profiler = GpuProfiler::new_with_tracy_client(
-            GpuProfilerSettings::default(),
+        #[cfg(not(any(target_os = "macos", target_os = "linux")))]
+        let mut profiler = GpuProfiler::new_with_tracy_client(
```

Comment:
I think we need to add this behind a proper feature flag, since the profiling works on Linux outside of the CI.